### ISSUE TITLE
thought i buit the plugins but i can only find "gain.vst3" and "gain.clap"

### ISSUE THREAD

**Issue by joshgura** [robbert-vdh/nih-plug#156]
I used the cargo command for rust that was written in the description. didn't see any errors. Am I supposed to find the built plugin suite somewhere else other than the cloned directory that i executed the rust command in?

never built a single plugin before so it was fun to at least build "gain.vst3" 

but I'm sure I'm doing something wrong. if there were pre-made plugins I'd just download them.

excited to try out some of these plugins.

**Comment by sout233** [comment]
Are you looking for nih plugins? You can download them at here: https://github.com/robbert-vdh/nih-plug/actions/workflows/build.yml?query=branch:master

just find the Artifacts in build page, and download it.

if you want to build them, just `cd` to `nih_plug/plugins/<the_plugin_u_want_to_build>`, run `cargo xtask bundle <PLUGIN_NAME> --release`. Then you can find vst3 and clap plugins in `target` folder.

**Comment by tphbrok** [comment]
Yes, building any plug-in in this repository only builds it in the `target` folder.

For your DAW to pick them up, you'd need to copy `gain.vst3` (or whichever plug-in or format you want) to the correct folder which your DAW looks in. For my set-up (macOS + Bitwig Studio), the target parent folder to copy to would be `/Library/Audio/Plug-ins/VST3`.

You can even combine everything in a single command if you wish (assuming you're in the plug-in folder itself, as @sout233 mentioned, and you're on the same set-up as me):

```sh
# Bundle for release
cargo xtask bundle gain --release && \

# Copy to the default VST3 plug-ins folder
sudo rm -rf /Library/Audio/Plug-ins/VST3/gain.vst3 && \
sudo cp -r target/bundled/gain.vst3 /Library/Audio/Plug-ins/VST3
```
